e2e/suites/actions/mark-favorite.test.ts

```diff
@@ -70,7 +76,8 @@ describe('Mark items as favorites', () => {
 
     afterAll(done => {
         Promise.all([
-            apis.user.nodes.deleteNodesById([ file1Id, file2Id, file3Id, file4Id, folder1Id ]),
+            apis.user.nodes.deleteNodesById([ file1Id, file2Id, file3Id, file4Id, folder1Id, fileSiteNotFav1Id, folderSiteId ]),
```

Comment:
no need to delete the files created in the site since you're deleting the site entirely